lib/WebDriver/ServiceFactory.php

```diff
@@ -103,6 +103,11 @@ public function setServiceClass($serviceName, $className)
             $className = '\\' . $className;
         }
 
+        // Flush outdated service cache
+        if (isset($this->serviceClasses[$serviceName]) && $this->serviceClasses[$serviceName] !== $className) {
+            unset($this->services[$serviceName]);
```

Comment:
Sorry, I understand what's going on now. I was skimming the code from my phone earlier.